README.md

```diff
@@ -28,50 +34,58 @@ This repository will configure the following items.
+Before running the bootstrap script, ensure that you have login access to your OpenShift cluster. This OpenShift cluster may have been provisioned though [RHDS](https://demo.redhat.com), or it may also be your own custom built cluster running on dedicated hardware.
+
+Make sure you are [logged into your cluster](https://docs.openshift.com/online/pro/cli_reference/get_started_cli.html) using the `oc login ...` command.  You can obtain a login token if required by utilizing the "Copy Login Command" found under your user profile in the OpenShift Web Console.
+
+The scripts require a user with sufficient permissions for installing and configuring operators, typically the `opentlc-mgr` user account on a Red Hat Demo System hosted cluster.
 
-The cluster will be created usually within an hour or so, and you should receive emails with links to access the cluster along with login credentials.
 
 ## Bootstrapping a Cluster
 
-Before beginning, make sure you are logged into your cluster using `oc login`. You can obtain a login token if required by utilizing the "Copy Login Command" found under your user profile in the OpenShift Web Console.
+Clone this git repository to a directory location on your local workstation.
 
-Next, clone this repository to your local environment.
+### Sealed Secrets
 
-### Sealed Secrets Bootstrap
+This repository deploys sealed-secrets and requires a sealed secret master key to during the bootstrap process. 
 
-This repository deploys sealed-secrets and requires a sealed secret master key to bootstrap.  If you plan to reuse sealed-secrets created using another key you must obtain that key from the person that created the sealed-secrets.
+If this is the first time you have run this script, you may need to add a Sealed Secret master key. If any of your gitops repos depend on Sealed Secrets you will need to place a copy of the key in `bootstrap/base/sealed-secrets-secret.yaml`. Note that the project currently does not yet deploy any Sealed Secrets so you can skip adding the sealed secret and instead ask the bootstrap script to create a master key for you.
 
-If you do not plan to utilize existing sealed secrets you can instead bootstrap a new sealed-secrets controller and obtain a new secret.
+The script will prompt: "Create NEW bootstrap/base/sealed-secrets-secret.yaml? [y/N]", you should answer:
 
-Execute the following script:
+| New? | Description |
+| ---- | ----------- |
+| Yes  | Choose this option if this is a brand new cluster. The bootstrap script will generate a new sealed secrets yaml file containing the master key generated bu the sealed-secrets controller |
+| No   | If your cluster already has a sealed secrets master key, you should copy it over to the bootstrap/base location. Also, if you've re-running the bootstrap script, this sealed secret file already exists. |
```

Comment:
| No | Choose this option if you already have Sealed Secrets you plan to deploy.  Instead, copy the Sealed Secrets master key into `bootstrap/base/sealed-secrets-secret.yaml` in order to allow you to unseal your Sealed Secrets. |
| No Prompt Given | If you have deployed this repo in the past, and your local copy of the repo already contains a  `bootstrap/base/sealed-secrets-secret.yaml` file, you will not be prompted to create a master key.  If you do not want to re-use a master key on the new cluster, consider removing the `bootstrap/base/sealed-secrets-secret.yaml` file before running the script.  |